vehicle/simvehicle.h

```diff
@@ -270,7 +270,10 @@ class vehicle_t : public vehicle_base_t, public test_driver_t
 	virtual bool check_next_tile(const grund_t* ) const OVERRIDE {return false;}
 
 public:
+	void change_route_index(sint16 i){route_index+=i;}
```

Comment:
`route_index` の外からの操作は `air_vehicle_t` に固有な理由で必要なので、`air_vehicle_t` に定義してください。使う側では`vehicle_t` を `air_vehicle_t` にキャストして呼びます。
```suggestion
	void increment_route_index(sint16 dx) { route_index += dx; }
```

Reply:
関数名ですが、`increment` のほうが良いと思います。`change`だと、`route_index`を引数の値に変更する、といった印象を受けます。